.circleci/config.yml

```diff
@@ -268,6 +271,123 @@ jobs:
                 include_job_number_field: false
                 include_project_field: false
                 failure_message: ':red_circle: WordPress Android connected tests failed on \`${CIRCLE_BRANCH}\` branch after merge by ${CIRCLE_USERNAME}. See <https://console.firebase.google.com/u/0/project/api-project-108380595987/testlab/histories/bh.e0c3a59bd9ed670|Firebase console test results> for details.\n\nPlease reach out in #platform9 if you think this failure is not caused by your changes, so we can investigate.'
+  raw-screenshots:
+    executor:
+      name: android/default
+      api-version: "28"
+    working_directory: /tmp/workspace
```

Comment:
This is needed because both executors of `raw-screenshots` and `promo-screenshots` needs to use the same `working_directory` in order for `attach_workspace` to work without issues. Without being explicit, the location of the home (default working dir) for Linux/Android executors and macOS executors being different, I've observed failure when trying to re-attach the workspaces.